pkg/platform/yaml/mockdb/db.go

```diff
@@ -133,6 +133,14 @@ func (ys *MockYaml) InsertMock(ctx context.Context, mock *models.Mock, testSetID
 	if err != nil {
 		return err
 	}
+
+	if exists, err := yaml.FileExists(ctx, ys.Logger, mockPath, mockFileName); err != nil {
+		utils.LogError(ys.Logger, err, "failed to find yaml file", zap.String("path directory", mockPath), zap.String("yaml", mockFileName))
+		return err
```

Comment:
No need of else-if, because if there is an error you should just return. and again check that if !exists then append.